validate_story.py

```diff
@@ -0,0 +1,106 @@
+""""Script to check story.json and story.py on missing references"""
+from config import Config
+import app.story.story as story_code
+
+import json
+import re
+
+
+def validate_story():
```

Comment:
should be in another MR
also, notice the change requests in https://github.com/EatingBacon/gameprog-detective-server/pull/78 regarding the validation